inst/@sym/ezplot.m

```diff
@@ -126,58 +135,66 @@
 end
 
 
+%!shared hf
+%! hf = figure ('visible', 'off');
```

Comment:
Ouch.  That's probably not worth it just so *I* don't see popup windows when I run local tests.

I'll revise and leave a pointer to https://savannah.gnu.org/bugs/?44478